### ISSUE TITLE
automatically report converted Math results

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5103]
**Reported by bhavyashah on 2015-05-23 10:12**
I propose automatically reading the mathematical results converted from the Math Type code.
I shall use a simple example to demonstrate my request more clearly:
When you want to type the fraction 1/2, you will type its Math Type code as $\frac{1}{2}$ and then press Alt + \ (back slash) to convert into a more readible format. I often desire to know if the converted mathematical form, which, in this case would be Math pause one halfpause space, immediately after pressing Alt + . To know this, I press the read current line command, but, due to wrong timing due to the conversion of code into readible Math being still under process, I often have to press the read current line command more than once.
A provision should be made in NVDA whic would make converted readible Math be read automatically.
Possible announcements could be the following:
Converted to Math {content}
Converted to {content}
or simply {content}
where {content} represents the readible mathematical content.
If some people dislike this proposal, perhaps a toggle option could be added, but, perhaps that would be an overkill?

**Comment by bhavyashah** [comment]
@feerrenrut @jcsteh Since this change would be a major relief for all NVDA+MathType users, I suggest labelling this ticket as P3, or even a P2 if implementation cost is not significant.

**Comment by derekriemer** [comment]
@bhavyashah we don't prioritize features.

**Comment by bhavyashah** [comment]
Right, this concept of mine was a little unclear a week ago. Ignore all other comments wherein I request the assignment of a priority to feature requests.

**Comment by bhavyashah** [comment]
@feerrenrut Not trying to push this ticket or anything, but as the original author of this ticket, please do let me know if there is any more information, any usage cases, or user impacts that you would like me to provide for you to better understand this issue.

**Comment by Adriani90** [comment]
cc: @NSoiffer do you have any suggestions to this one?
I must say I don't really use these conversion methods that often but maybe it is very common among power math users.
What do you think?

**Comment by seanbudd** [comment]
@bhavyashah - is this issue still relevant to you?
if so, can you report it to Microsoft so that they fire a UIA event
